### DISCUSSION TITLE
Another case of really early transition

### DISCUSSION THREAD

**Discussion by rainman110** [jomjol/AI-on-the-edge-device#2742] [Q&A]
I have a watermeter that starts to transition the last digit, when the first analog meter is between 3 and 4.

The postprocessing cannot correct for this, as the earliest transition possible is 6.0.

The first image correctly recognizes the digits.  The last digit (1) just starts with the transitioning, but it is obviously a 1.
![Screenshot_20231213_144325_com android chrome](https://github.com/jomjol/AI-on-the-edge-device/assets/3213107/a54bf445-3d4a-4683-8746-99bc47dd6f7d)

On the next image, about 100 liters have been used, so the first analog meter moved from 3 to 4. This is when the last digit switches already from 1 to 2.
![Screenshot_20231213_200937_com android chrome](https://github.com/jomjol/AI-on-the-edge-device/assets/3213107/9044aba6-490a-4b7f-8286-a35dbee3e91b)

Is there a way to configure this or workaround this issue?

**Comment by gintonic26** [comment]
Same Watermeter with the same Problem here.

**Comment by SybexX** [comment]
https://github.com/jomjol/AI-on-the-edge-device/issues/3025
